style.css

```diff
@@ -243,13 +255,15 @@ main {
 
 .skillcards-container {
   padding-top: 4rem;
-  gap: 2rem;
+  justify-items: center;
+  gap: 1.5rem;
+  flex-wrap: wrap;
```

Comment:
- [ ] **[OPTIONAL]** Please make sure the size of these cards stays similar to the figma design even when the screen is resized. You can do that my adding a `max-width` or use a flex-basis. This will make your project look way prettier 😄 🌹

**Your project**
![image](https://user-images.githubusercontent.com/74411135/228976182-a79f7153-84cf-4689-816f-a28054dba4d7.png)

**The figma design**
![image](https://user-images.githubusercontent.com/74411135/228976493-8522c9e8-ca21-48f4-805c-4c22f3e34cb8.png)